### ISSUE TITLE
Discoloration and Stuck Divider of Tabs

### ISSUE THREAD

**Issue by LookingForWayOut** [tabkit/tabkit2#84]
I have had this issue for many, many months. Since last summer, on two desktops. One at home running Vista and one running Windows 7 Enterprise, which was a fresh install all around last winter and the first install of Firefox and TabKit2 at the time has the same issues as my desktop since the summer (or even earlier, I do not recall).

I got two separate issues no matter what I do in settings. One, unrelated sites sharing colors of other groups, or not having the same color as a group of others that do. Two, a divider bar with some sort of split of shade.

See these images please: https://www.mediafire.com/folder/ooxbursb12igd/tabkit
untitled.png - See how half of it is darker than the other? It is like a permanent shadow. When I scroll, that shadow, that divider, stays. You can also see how two different websites share the same color as well.
untitled.jpg - That divider border again. Hard to even read the tabs under the bar.
tabkit.jpg - See on the right how the YouTube videos are two different colors, and a Google result is the same as YouTube?

Are there logs I can grab for you, or a resolution?

Thank you.

**Comment by PikachuEXE** [comment]
For the first issue (shadow/divider), it could be the issue of transparent background of Win7.
Try installing https://addons.mozilla.org/en-US/firefox/addon/tab-kit-tab-highlighter/ and changed some config.

For the second issue, it would be easier to find out the cause if you can reproduce with [Mozilla Firefox, Portable Edition](http://portableapps.com/apps/internet/firefox_portable) and send me the whole folder (zipped) so I can understand the dependencies & settings you are using without typing a long text here.

**Comment by conquerist** [comment]
I think that the first issue is related to the Window Classic theme. I can replicate it if I set my theme to Windows Classic (Windows 7, FF 37.0.1) and do the following:

1. Clean profile
2. Set Windows theme to "Windows Classic"
3. Install CTR 1.3.0 and Tabkit2 0.12.2, restart FF
4. Enable the menu bar to fix the Firefox layout (seems to be necessary with Windows Classic theme?)
5. Make window smaller than full screen, maximize the window, and restore the window. This step may not be necessary, depending on whether the Firefox Window has been maximized in the current session.
6. Increase window size by dragging bottom right window corner. The newly uncovered part of the tab pane will have a gray background, the previously covered part a blue background.

Without CTR and with coloring set to "Color labels by group...", the coloring will be painted on top of the tabs.

Will edit with screenshots in a minute.

**Comment by LookingForWayOut** [comment]
Pik, thanks for writing back. I will try that portable version and reply with results for you.

For this addon you provided, the only change I get, other than tabs being yellow or black (for current) was that disabling the "make tab bar's background solid" made everything the greyish color (as seen in first image), but obviously hard to see the white text of the tabs. Think will need another addon for that. So now, this has shown me it has something to do with the bar's background.

**Comment by LookingForWayOut** [comment]
Conquerist, I see your post. Thank you. Will try that as well and report my findings. I can probably do it on the W7 today but the Vista has to wait until tonight.

**Comment by LookingForWayOut** [comment]
Conquerist, what you suggested work for this W7 unit. The background is one color, tabs are grey with black text. Doable. Thank you.
Pik, I was going to try the portable version method at home with my Vista machine, but you will not believe what I am going to say... all of the sudden, the tab bar looks fine. I did not touch it at all. Tab kit not even updated to latest version and yet everything looks as it does without touching a thing. For months and months it has driven me crazy, even the other week, and when I went home, it looked fine. Maybe it knew to be afraid? I really have no idea how it looks normal now and nothing has changed in months on it. Very interesting...
If anything changes on either of these two machines, I shall keep you posted (no pun).
Thank you very much.
